swagger.yml

```diff
@@ -0,0 +1,64 @@
+swagger: '2.0'
+basePath: /
+paths:
+  /:
+    get:
+      responses:
+        '200':
+          description: Success
+      summary: Endpoint for Liveliness
+      operationId: get_liveliness
+      tags:
+        - default
+  /api/v0/publish:
+    post:
+      responses:
+        '200':
+          description: Success
```

Comment:
Does Swagger allow us to describe the response structure? I mean, it would be nice, if we can describe the `status`, `message` and more importantly the `error` properties here as well. It can facilitate testing the API.

Reply:
Yes, it does.
If you use Swagger API libraries like `connexion`, the response structure would be the same as defined in swagger.yml.

Reply:
I'm not sure if I fully understand. But it actually doesn't matter much. Having the response fields described in great detail would ne a nice to have feature, but not a necessity. I think the current state of `swagger.yml` is good enough. And if we want we can further improve it for sure... So. Let's start with this and coordinate with QE if there's any further work to be done.